web-portal/package.json

```diff
@@ -17,6 +17,12 @@
     "generate": "nuxt generate"
   },
   "dependencies": {
+    "@fullcalendar/core": "^5.11.3",
+    "@fullcalendar/daygrid": "^5.11.3",
+    "@fullcalendar/interaction": "^5.11.3",
+    "@fullcalendar/moment": "^5.11.3",
```

Comment:
So I gather we need the `interaction` plugin for clicking on events, but do we actually need the `moment` plugin?

Reply:
@MatthewGidcomb yes, it is an extra plugin, not necessary to use it now. Removed it 25b402e40d42d2098a0f02b28bfa64ffcbc8a962